hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -1257,6 +1257,35 @@ Y(incwm, incw, loadw, storew, m)
 
 #undef Y
 
+void lower(Vunit& unit, cvttsd2siq& i, Vlabel b, size_t idx) {
+  lower_impl(unit, b, idx, [&] (Vout& v) {
+  auto fpsr = v.makeReg();
+  auto res = v.makeReg();
+  auto const sf = v.makeReg();
+  auto err = v.makeReg();
+  auto tmp1 = v.makeReg();
+  auto tmp2 = v.makeReg();
+
+  // Clear FPSR IOC flag
+  v << mrs{FPSR, tmp1};
+  v << andqi{~0x01, tmp1, tmp2, sf};
+  v << msr{tmp2, FPSR};
```

Comment:
Thanks for the review.
AFAIK, there should not be any extra penalty other than the usual cost of executing the register copy. It is possible to just write zero to FPSR instead of read-modify-write to clear the IOC bit (bit 0) alone.